ImgSmall.java

```diff
@@ -0,0 +1,65 @@
+import javax.imageio.ImageIO;
+import java.awt.Image;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.file.Paths;
+
+public class ImgSmall {
+  public static void main(String[] args) throws IOException {
+      resize(args);
+  }
+  public static void resize(String[] args) throws IOException{
+    if (args.length != 2){
+      System.out.println("please put in a url and a percent");
+      System.exit(0);
+    }
+    //get url object
+    URL url = null;
+    try{
+      url = new URL(args[0]);
+    } catch (MalformedURLException mfue){
+      System.out.println("check the url");
+      System.out.print(args[0]);
+      System.exit(0);
+    }
+    double resizeBy = (100 - Integer.parseInt(args[1])) / 100.0;
```

Comment:
What about the error if someone enters arg[1] as 101 (>= 100)
You should check the constraints of the arguments and give back meaningful errors to the user